### DISCUSSION TITLE
CAPI OKE on free tier

### DISCUSSION THREAD

**Discussion by stenwt** [oracle/cluster-api-provider-oci#390] [Q&A] [answered]
I've tried using [cluster-template-managed](https://github.com/oracle/cluster-api-provider-oci/blob/main/templates/cluster-template-managed.yaml) and [cluster-template-managed-private](https://github.com/oracle/cluster-api-provider-oci/blob/main/templates/cluster-template-managed-private.yaml) to create an OKE cluster via Cluster API on the free tier. In both cases, the create failed with: `failed to reconcile NAT Gateway: failed create nat gateway: Error returned by VirtualNetwork Service. Http Status Code: 400. Error Code: LimitExceeded`. Is there a workaround to this where I could deploy an OKE cluster on the free tier? I was looking to use this as a demo environment to show some of my coworkers the value of CAPI.

**Comment by joekr** [comment, marked as answer]
It sounds like maybe you already have a NAT gateway setup? I believe you only get one for the trial. If that is the case it may be possible to use the same NAT Gateway, but I've not tested that. You should be able to set the NAT gateway id in the VCN https://github.com/oracle/cluster-api-provider-oci/blob/main/templates/cluster-template-managed-private.yaml#L29 and set a NAT gateway with the existing id https://github.com/oracle/cluster-api-provider-oci/blob/14936bf82f337b824dfec69591c50b97c6f0bdb5/api/v1beta2/types.go#L1125

**Comment by stenwt** [comment]
Thanks, I haven't been able to find it but I deleted my account and created a new one and was able to build a cluster.
